src/color.ts

```diff
@@ -103,15 +103,16 @@ export function hslToRGBA(hsl: number[]): number[] {
     rgb = [x, 0, c];
   } else if (h < 360) {
     rgb = [c, 0, x];
+  } else {
```

Comment:
@xiaoxiangmoe 

It's unreachable code.

Reply:
If I don't write this, it will throw error.